app/assets/stylesheets/templates/_ctc-home.scss

```diff
@@ -18,10 +18,34 @@
     padding-bottom: 0;
   }
 
-  .slab--why {
+  .slab--affiliations {
     background-color: $color-grey-light;
   }
 
+  .affiliations {
```

Comment:
What happens to this section on mobile? Do the sections stack on top of one another? I can't tell if there is something that changes the flex direction on a parent class or something.

Reply:
oh, now that im looking at how its implemented, maybe the width one third class handles that?